cobalt/dom/source_buffer.cc

```diff
@@ -521,6 +521,21 @@ void SourceBuffer::TraceMembers(script::Tracer* tracer) {
   tracer->Trace(video_tracks_);
 }
 
+size_t SourceBuffer::memory_limit(
+    script::ExceptionState* exception_state) const {
+  if (memory_limit_ == 0) {
+    web::DOMException::Raise(web::DOMException::kInvalidStateErr,
+                             exception_state);
+  }
+  return memory_limit_;
+}
+
+void SourceBuffer::set_memory_limit(size_t memory_limit,
+                                    script::ExceptionState* exception_state) {
+  memory_limit_ = memory_limit;
```

Comment:
Consider raising `kNotSupportedErr` when `memory_limit` is 0.

Reply:
done.